scvi/module/_amortizedlda.py

```diff
@@ -0,0 +1,349 @@
+import math
+from typing import Dict, Iterable, Optional, Sequence, Tuple, Union
+
+import pyro
+import pyro.distributions as dist
+import pyro.poutine as poutine
+import torch
+import torch.nn.functional as F
+from pyro.infer import Trace_ELBO
+from pyro.nn import PyroModule
+
+from scvi._constants import _CONSTANTS
+from scvi.module.base import PyroBaseModuleClass, auto_move_data
+from scvi.nn import Encoder
+
+_AMORTIZED_LDA_PYRO_MODULE_NAME = "amortized_lda"
+
+
+class CategoricalBoW(dist.Multinomial):
+    def log_prob(self, value):
+        if self._validate_args:
+            self._validate_sample(value)
+        logits, value = dist.util.broadcast_all(self.logits, value)
+        logits = logits.clone(memory_format=torch.contiguous_format)
+        logits[(value == 0) & (logits == -math.inf)] = 0
+        log_powers = (logits * value).sum(-1)
+        return log_powers
+
+
+def logistic_normal_approximation(
+    alpha: torch.Tensor,
```

Comment:
can you add a comment/docstring about what's going on here?

Reply:
Also realized that the sigma here was the variance when I should be returning the std, don't think it actually changes a lot though